### PR TITLE
Adding start, step for randint and randchoice

### PR DESCRIPTION
**Pull request by driesceuppens**
### Summary
Adding start and step for the `randint()` function in aliasing and the `randchoice()` function.

#### PR Type

- [x] This PR is a code change that implements a feature request.
- [ ] This PR fixes an issue.
- [ ] This PR adds a new feature that is not an open feature request.
- [ ] This PR is not a code change (e.g. documentation, README, ...)
#### Other
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] If code changes were made then they have been tested.
- [x] I have updated the documentation to reflect the changes.

### CODE REVIEW

**Comment by driesceuppens** [comment]
If needed I could also run some tests for the time it takes with timeit.

File: tests/aliasing/statblock_test.py [hunk 1]
```diff
@@ -1,45 +0,0 @@
-from aliasing.api.statblock import AliasSkill
```
Comment (zhudotexe) [inline tests/aliasing/statblock_test.py:1]: Why was this file deleted?

Comment (driesceuppens) [reply tests/aliasing/statblock_test.py:1]: I renamed the test file from `statblock_test` to `functions_test` since that made more sense.

File: tests/aliasing/functions_test.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+from aliasing.api.functions import randint, randchoice
+import pytest
+
+
+# TODO: AliasStatBlock tests
+# TODO: AliasBaseStats tests
+# TODO: AliasLevels tests
+# TODO: AliasAttack tests
+# TODO: AliasSkill tests
```
Comment (1drturtle) [inline tests/aliasing/functions_test.py:9]: remove these TODOs since they are already in the statblock tests file

File: tests/aliasing/functions_test.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        
+        assert r3 >= 10
+        assert r3 % 2 == 0
+        assert r3 < 100
+        
+        assert c1 in seq
+# TODO: AliasSaves tests
+# TODO: AliasResistances Tests
+# TODO: AliasSpellbook Tests
+# TODO: AliasSpellbookSpell tests
```
Comment (1drturtle) [inline tests/aliasing/functions_test.py:33]: same as above with these

File: aliasing/api/functions.py [hunk 4]
```diff
@@ -156,5 +156,13 @@ def rand():
...
-    return random.randrange(top)
+def randint(start, stop=None, step=None):
+    if stop is None and step is None:
+        return random.randrange(start)
+    elif stop is not None and step is None:
+        return random.randrange(start, stop)
+    return random.randrange(start, stop, step)
+
+
+def randchoice(seq):
```
Comment (zhudotexe) [inline aliasing/api/functions.py:167]: Defining this here won't expose it in the Draconic context - you should add to to `aliasing.evaluators.DEFAULT_BUILTINS` too

Comment (driesceuppens) [reply aliasing/api/functions.py:167]: Should I keep `randint` in the legacy part or move it to ours?
